cli/src/test/java/com/devonfw/tools/ide/tool/intellij/IntellijTest.java

```diff
@@ -110,35 +109,14 @@ public void testIntellijPluginInstallWithCustomRepoUrl(String os) {
     Intellij commandlet = new Intellij(this.context);
 
     // act
-    commandlet.install();
+    commandlet.run();
```

Comment:
IMHO this test change should not be needed anymore.
If it is needed we should check why and most probably it indicates a bug produced by this PR.

Reply:
Adjusted to install.